.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,68 @@
+name: 🐛 Bug Report
+description: File a bug report
+title: "[BUG]: "
+labels: ["bug", "triage", "help-wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: What version of our software are you running?
+      placeholder: "e.g. v1.0.0"
+    validations:
+      required: true
+  
+  - type: dropdown
+    id: environment
+    attributes:
+      label: Environment
+      description: Where are you running our software?
+      options:
+        - JupyterHub
+        - Local
+        - Other
+    validations:
+      required: true
+      
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+      
+  - type: textarea
+    id: steps
+    attributes:
+      label: Steps to Reproduce
+      description: How can we reproduce this issue?
+      placeholder: |
+        1. Go to '...'
+        2. Click on '....'
+        3. Scroll down to '....'
+        4. See error
+    validations:
+      required: true
+  
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code.
+      render: shell
+      
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      description: By submitting this issue, you agree to follow our Code of Conduct
+      options:
+        - label: I agree to follow this project's Code of Conduct
+          required: true
```

Comment:
I like this in theory but we don't have a Code of Conduct for our repository yet as far as I'm aware. We have a guide for contributors, but I wouldn't call it a code of conduct necessarily. Also I think if we have this requirement (which I think is a cool idea), we should also provide in the same area a link to the code of conduct.

Reply:
Yeah, for now I think we can remove this, but I would like to revamp our contribution guidance and add one.